src/utils/misc.ts

```diff
@@ -1 +1,3 @@
+export type Optional<T> = T | undefined // I'm pretty sure this exists somewhere and can't find it
```

Comment:
Where do we need this

Reply:
If this is used in an object type can we not just use 
```ts
key?: type
```

Reply:
You're right, I never used it. I was meaning to use it in the `siblingCourseId` type. I'll get rid of the definition. Watch this video though it's poggers:

[https://www.youtube.com/watch?v=C2w45qRc3aU](https://www.youtube.com/watch?v=C2w45qRc3aU)